### PR TITLE
Completed HTML-Block-and-Inline - Macy Hong

### PR DESCRIPTION
**Pull request by macyhongg**
image

### CODE REVIEW

File: html-block-and-inline/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+<h1>Blue Americana Snuggie&reg; Blanket</h1>
+<h2>by SnuggieStore</h2>
+<p>Price: <s>$19.99</s> $14.99</p>
+<p>Part of our new Americana collection, this <em>beautiful</em> blue is already a customer favorite!</p>
+<p>Enjoy complete <strong>Freedom of Movement!</strong></p>
+<p>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</p>
```
Comment (BrettAlbright) [inline html-block-and-inline/index.html]: make sure that all of these lines are properly indented as children of the `body` tag

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good, just some indenting to update

File: html-block-and-inline/index.html (hunk 1 above)
Comment (macyhongg) [reply html-block-and-inline/index.html]: Would p elements be indented from header elements as well?

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**